typescript/introduction/tsc-using-types/tsc-number-string-boolean.md

```diff
@@ -0,0 +1,89 @@
+---
+author: kapnobatai136
+type: normal
+category: must-know
+aspects:
+  - introduction
+links:
+  - '[Number Type](https://www.typescriptlang.org/docs/handbook/basic-types.html#number){documentation}'
+  - '[String Type](https://www.typescriptlang.org/docs/handbook/basic-types.html#string){documentation}'
+  - '[Boolean Type](https://www.typescriptlang.org/docs/handbook/basic-types.html#boolean){documentation}'
+
+---
+
+# Number, String & Boolean
+
+---
+## Content
+
+We can define a type for a variable by adding a semi-colon (`:`) and a type after its name.
```

Comment:
I'd change `define` to `attach` or something similar. 

When I hear "define a type" I think about creatinig a type alias / interface, like a variable